src/py/reactpy/reactpy/core/hooks.py

```diff
@@ -248,6 +263,16 @@ def use_context(context: Context[_Type]) -> _Type:
     return provider.value
 
 
+def use_scope() -> MutableMapping[str, Any]:
+    """Get the current :class:`~reactpy.backend.types.Connection`'s scope."""
+    return use_connection().scope
+
+
+def use_location() -> Location:
+    """Get the current :class:`~reactpy.backend.types.Connection`'s location."""
+    return use_connection().location
+
+
```

Comment:
Minor nitpick, but could we move these hooks all be next to each other?

Right now, `use_connection` isn't sitting next to the rest of these hooks.